docs/api.rst

```diff
@@ -4702,9 +4703,81 @@ Dealing with History Events
           "location_label": "0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12",
           "notes": "Approve 1 SAI of 0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12 for spending by 0xdf869FAD6dB91f437B59F1EdEFab319493D4C4cE",
           "event_subtype": "approve",
-          "counterparty": "0xdf869FAD6dB91f437B59F1EdEFab319493D4C4cE"
+          "counterparty": "0xdf869FAD6dB91f437B59F1EdEFab319493D4C4cE",
+          "extra_data": {}
+      }
+
+   .. http:example:: curl wget httpie python-requests
+
+      PUT /api/1/history/events HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {
+          "entry_type": "history event",
+          "event_identifier": "RE_xxxxxxxxxx",
+          "location": "ethereum",
+          "timestamp": 1569924574,
+          "balance": {"amount": "1.542", "usd_value": "1.675"},
+          "sequence_index": 162,
+          "event_type": "informational",
+          "event_subtype": "approve",
+          "asset": "eip155:1/erc20:0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359",
+          "location_label": "0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12",
+          "notes": "Approve 1 SAI of 0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12 for spending by 0xdf869FAD6dB91f437B59F1EdEFab319493D4C4cE"
+      }
```

Comment:
I'm not sure but is it possible to create sub sections inside this section for the different event types and then have each one with their special fields only and then later the fields that are common for all with a reference? I believe it would improve this section a lot